### PR TITLE
Add necessary dependencies for ignition-rendering.

### PR DESCRIPTION
**Pull request by mahiuchun**

### CODE REVIEW

File: ignition-rendering/PKGBUILD [hunk 1]
```diff
@@ -8,8 +8,8 @@ engines. It offers unified APIs for creating 3D graphics applications."
 arch=('x86_64')
 url="https://ignitionrobotics.org/libs/rendering"
 license=('Apache')
-depends=('ignition-plugin')
-makedepends=('cmake')
+depends=('ignition-common' 'ignition-math' 'ignition-plugin' 'ogre-1.9')
```
Comment (acxz) [inline ignition-rendering/PKGBUILD:11]:
```suggestion
depends=('ignition-common' 'ignition-math' 'ignition-plugin' 'ogre=1.9')
```

Just a little bit cleaner.

**Review by acxz — CHANGES_REQUESTED** [review]
Can you also bump the `pkgrel'?

See: https://wiki.archlinux.org/title/PKGBUILD#pkgrel

File: ignition-rendering/PKGBUILD (hunk 1 above)
Comment (mahiuchun) [reply ignition-rendering/PKGBUILD]: I went with `=` as you suggested as ignition-rendering would not build with Arch’s `ogre` package yet.

**Comment by mahiuchun** [comment]
Done

**Pull request merged**